server/views/pages/overview.njk

```diff
@@ -328,6 +330,51 @@
           {% endif %}
         </div>
       </section>
+
+    {% if uploadDocumentsEnabled %}
```

Comment:
Not a massive issue but it looks like this bit has 4 spaces per indentation but the rest of the file is 2 spaces

Reply:
This bit is autoformatted by prettier, I didn't want to format the whole file. I can if you want though

Reply:
I'll raise a tech debt task to sort out njk formatting in general.